.github/actions/publish-nuget/action.yml

```diff
@@ -154,7 +155,8 @@ runs:
             "-p:WarnOnPackingNonPackableProject=True",
             "-p:IncludeSymbols=${{ inputs.dotnet-pack-include-symbols }}",
             "-p:SymbolPackageFormat=snupkg",
-            "-p:NoDefaultExcludes=true"
+            "-p:NoDefaultExcludes=true",
+            "-p:EnablePackageValidation=true"
```

Comment:
No baseline version validation here(as you mentioned under the issue) but under the related [wiki page](https://lombiq.atlassian.net/wiki/spaces/DEV/pages/786857987/Managing+releases+of+open-source+projects#:~:text=To%20check%20for%20breaking%20changes%20in%20the%20.) the `-p:PackageValidationBaselineVersion=$current` is present.
Why the difference?

Reply:
See https://github.com/Lombiq/GitHub-Actions/issues/45#issuecomment-1240568296

Reply:
I saw your comment under #45  and I understand why `p:PackageValidationBaselineVersion` isn't here, my question is why it is required under the wiki.
But now it's clear. (I read the wiki again)